src/dialects/abstract/query-generator-typescript.ts

```diff
@@ -39,6 +39,14 @@ export class AbstractQueryGeneratorTypeScript {
     return this.sequelize.options;
   }
 
+  describeTableQuery(table: TableNameWithSchema) {
```

Comment:
`TableName` was the right type here. It's equal to `TableNameWithSchema | string`, the string version gets normalized into a `TableNameWithSchema` by extractTableDetails in quoteTable

We could even accept `TableNameOrModel`, eventually all(?) queryGenerator methods will start accepting models I think

Reply:
Yeah, I tried delegating the extraction to `describeTable` so I could send the deprecation warnings there. But it's nicer to just do it here since in some dialects we use `quoteTable` so it's done double. I should have updated it to `TableNameOrModel` now